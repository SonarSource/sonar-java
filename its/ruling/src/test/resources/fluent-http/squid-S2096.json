{
'net.code-story:http:src/main/java/net/codestory/http/misc/ExtractWebjars.java':[
27,
],
}
