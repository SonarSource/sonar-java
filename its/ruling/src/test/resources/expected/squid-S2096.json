{
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
2003,
],
'project:com/sun/org/apache/xml/internal/security/keys/storage/implementations/CertsInFilesystemDirectoryResolver.java':[
203,
],
'project:com/sun/org/apache/xml/internal/security/keys/storage/implementations/KeyStoreResolver.java':[
127,
],
'project:com/sun/org/apache/xml/internal/security/utils/RFC2253Parser.java':[
501,
],
'project:com/sun/security/auth/PolicyParser.java':[
957,
],
}
